configure

```diff
@@ -600,10 +600,14 @@ int hello() {return getchar();}
 EOF
 if test $shared -eq 1; then
   printf "Checking for shared library support... " | tee -a configure.log
+  # Workaround for lld not liking --version-script
+  LDDSHAREDFLAGS="$(echo $LDSHAREDFLAGS | sed 's/,--version-script,[^ ]\+//')"
```

Comment:
I would not use sed to remove linker flags. It fails on: `-Wl,--version-script,...`, leaving `-Wl`, causing the compiler to complain about an unrecognized flag.

Reply:
It's supposed to fail if `--version-script` is specified alone... In zlib-ng's configure script it is never specified alone.